app/controllers/users_controller.rb

```diff
@@ -17,9 +22,42 @@ def create
       render :new
     end
   end
+  def edit
+    @user = User.find_by(id: params[:id])
+  end
+  def update
+    @user = User.find_by(id: params[:id])
+    if @user.update(user_params)
+      flash[:success] = "Profile updated"
+      redirect_to @user
+    else
+      render "edit"
+    end
+  end
+  def destroy
+    user = User.find_by(id: params[:id])
+    if user&.destroy
```

Comment:
`&` ko cần toán tử an toàn nhé

vì dùng method `load_user` em đã handle việc user bị nil ở load_user rồi